data/ka'het/ka'het missions.txt

```diff
@@ -99,56 +120,45 @@ mission "First Contact: Ka'het: Remnant 1B"
 	planet Celeatis
-		spaceport `This planet appears to have been home to one of the largest Builder research centers, which focused on studying the anomaly in the system. Had they found out how to stabilize it and use it as a wormhole, perhaps their species would have survived and spread to the rest of the Ember Waste.`
-		spaceport `	Due to the infertile and rugged terrain, the inhabitants of Celeatis had to rely on the food produced abroad. When the Ka'het came, the supplies abruptly stopped arriving, and in less than a year the planetary food reserves were depleted. The planet's population who had survived the invasion didn't last much longer.`
+		description `	This planet appears to have been home to one of the largest Builder research centers, which focused on studying the anomaly in the system. Had they found out how to stabilize it and use it as a wormhole, perhaps their species would have survived and spread to the rest of the Ember Waste.`
 	planet Ceilia'sei
-		spaceport `Celia'sei was constructed to act as some sort of anti-missile platform. When the Ka'het invasion broke out for the second time, it must have been able to shoot down the EMPs fired nearby without any trouble, but it had no way to defend itself against the deadly Ravager Beams. The crew quarters were soon blown up by the incoming fleets, and the station's defense systems were subsequently lost forever.`
-		spaceport `	Given its location, it's likely that Ceilia'sei was originally built to stop something coming out of the wormhole rather than the Ka'het themselves.`
+		description `	The main purpose of Celia'sei to act like some sort of anti-missile platform. But when the Ka'het invasion broke out for the second time, it had no way to defend deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
 	planet Eavine
-		spaceport `Eavine played a crucial role in the demise of the Builders: here, a second wave of Ka'het was built to push the first one back. They soon found out that what was supposed to stop the invasion only accelerated it: the new Ka'het refused to obey the Builders and allowed their brethren to reach the northern systems.`
-		spaceport `	The moon had been terraformed to make it a more appealing place to live, but after their civilization collapsed the moon began to return to its natural state. A few thousand years ago, Eavine fully reverted to its original sub-zero temperatures. The facilities stopped working, and the Ka'het production in this site came to an end.`
+		description `	This moon was also home for the second Ka'het wave, refusing to obey again and for the last time. This moon had been terraformed to make it a more appealing place to live, but after their civilization collapsed the moon began returned to its natural state. When it reverted to its original sub-zero temperatures, the facilities stopped working, and the Ka'het production in this site came to an end.`
```

Comment:
```suggestion
		description `	This moon was also home for the second Ka'het wave, which, same as the first, refused to obey their creators. It had been terraformed to make it a more appealing place to live, but after their civilization collapsed the moon began returned to its natural state. When it reverted to its original sub-zero temperatures, the facilities stopped working, and the Ka'het production in this site came to an end.`
```
I'll be honest: I don't remember all of the lore of the Builders and the Ka'het off the top of my head, so hopefully this change is still historically correct. Let me know if it isn't.

Reply:
It is